pkgs/by-name/ll/lldap/package.nix

```diff
@@ -3,29 +3,31 @@
   fetchFromGitHub,
   lib,
   lldap,
+  makeWrapper,
   nixosTests,
   rustPlatform,
   rustc,
-  wasm-bindgen-cli_0_2_95,
+  wasm-bindgen-cli_0_2_100,
   wasm-pack,
   which,
 }:
 
 let
+  version = "0.6.2";
 
-  commonDerivationAttrs = rec {
+  commonDerivationAttrs = {
     pname = "lldap";
-    version = "0.6.1";
+    inherit version;
 
     src = fetchFromGitHub {
       owner = "lldap";
       repo = "lldap";
       rev = "v${version}";
-      hash = "sha256-iQ+Vv9kx/pWHoa/WZChBK+FD2r1avzWWz57bnnzRjUg=";
+      hash = "sha256-UBQWOrHika8X24tYdFfY8ETPh9zvI7/HV5j4aK8Uq+Y=";
     };
 
-    cargoHash = "sha256-qXYgr9uRswuo9hwVROUX9KUKpkzR0VEcXImbdyOgxsY=";
-
+    useFetchCargoVendor = true;
```

Comment:
Looks like this option is gone now, and the resulting warning breaks the review builds: https://github.com/NixOS/nixpkgs/actions/runs/17053227263/job/48345573590?pr=425918#step:6:258

Reply:
I did look at the build issues but must have missed the root cause. Will take care of that.